src/breadcrumbs/breadcrumbs.scss

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+@mixin iui-breadcrumbs {
+  @include iui-reset;
+  @include iui-font-family;
+  display: flex;
+  align-items: center;
+  
+  .iui-breadcrumbs-list {
+    @include iui-reset;
+    list-style-type: none;
+    display: flex;
+    align-items: center;
+    user-select: none;
+
+    .iui-breadcrumbs-item a,
+    .iui-breadcrumbs-item .iui-button .iui-label {
+      overflow: hidden;
+      white-space: nowrap;
+      text-overflow: ellipsis;
+      max-width: $iui-3xl * 2;
+      text-align: left;
+    }
+
+    .iui-breadcrumbs-item {
+      .iui-icon {
+        vertical-align: middle;
```

Comment:
I would have expected flex is used maybe for the breadcrumb item and centering things...

Reply:
This can be removed now with the changes I made in the last push fixing the vertical alignments of the crumbs.